### DISCUSSION TITLE
Allow namespace selection by label selectors

### DISCUSSION THREAD

**Discussion by pantosha** [emberstack/kubernetes-reflector#409] [Ideas]
Now `reflector.v1.k8s.emberstack.com/reflection-allowed-namespaces` and `reflector.v1.k8s.emberstack.com/reflection-auto-namespaces` support only names of namespaces.
k8s has the concept of [labels and label selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors).

It would be nice to support label selectors in reflector, e.g. add new annotations:
* `reflector.v1.k8s.emberstack.com/reflection-allowed-namespaces-selector: label-selector`
* `reflector.v1.k8s.emberstack.com/reflection-auto-namespaces-selector: label-selector`

Label selectors are supported by [kubernetes-replicator](https://github.com/mittwald/kubernetes-replicator), [config-syncer](https://config-syncer.com/docs/v0.14.6/guides/config-syncer/intra-cluster/)

**Comment by davidswimbird** [comment]
This has been implemented as part of [this](https://github.com/emberstack/kubernetes-reflector/pull/620) PR. Included in version 10.0.38 of Reflector.
